packages/base-styles/_colors.scss

```diff
@@ -25,6 +25,14 @@ $light-gray-200: #f3f4f5;
 $light-gray-100: #f8f9f9;
 $white: #fff;
 
+// G2 colors.
```

Comment:
I have the same remark about these variables. I think we should avoid using "gray", "blue" in these. Even "dark", "light" I'm not sure about them.

I'd prefer if we use semantic variables. "primary-border-color"... That way when we decide to change the color itself, we won't have to change the variable name.

Reply:
I see that point. @pablohoneyhoney any thoughts?

Reply:
I think we should change these variable names.